src/player.cpp

```diff
@@ -147,12 +147,12 @@ void Player::Init(int argc, char *argv[]) {
 	if (!addtl_ver.empty())
 		header << " " << addtl_ver;
 	header << " started";
-	Output::Debug(header.str().c_str());
+	Output::Debug(header.str());
```

Comment:
Not your fault but because I see it: Can you change this to ``{}, header`` so we don't have a variable in the first argument?